comparison.js

```diff
@@ -0,0 +1,9 @@
+function strictEqual(var1, var2) {
+    if ( (var1 == var2) && (typeof var1 == typeof var2) ) {
+        return true 
+    } 
+
+    return false
+}
```

Comment:
Nice solution. Notice though that we're saying if the condition is true return true and if the condition is false return false. That means we can just return the condition.

```JavaScript
function strictEqual(var1, var2) {
    return var1 == var2 && typeof var1 == typeof var2
}
```

Since this is a one line return function we can also switch to ES6+ syntax and take advantage of fat arrow's implicit return

```JavaScript
const strictEqual = (var1, var2) => var1 == var2 && typeof var1 == typeof var2
```